src/collection-facets.ts

```diff
@@ -401,10 +401,11 @@ export class CollectionFacets extends LitElement {
                     />
                     <label
                       for=${negativeCheckboxId}
-                      class="hide-facet-icon"
+                      class="hide-facet-icon ${facetHidden ? 'active' : ''}"
```

Comment:
Suggestion: Allow for space differently. Example: 

`class="hide-facet-icon${facetHidden ? ' active' : ''}"`